finesse/hardware/plugins/spectrometer/opus_interface.py

```diff
@@ -122,6 +123,10 @@ def _on_reply_received(self, reply: QNetworkReply) -> None:
 
         # If the status has changed, notify listeners
         if new_status != self._status:
+            # On first update, we need to signal that the device is now open
+            if self._status == SpectrometerStatus.UNDEFINED:
```

Comment:
Isn't it a bit odd that when it's undefined, it is set as open?

Reply:
Sorry, to clarify -- I set it to UNDEFINED in `__init__` before we actually have a status, so it's a sentinel value. But it's probably not a good idea, for the reason I mentioned